cmd/query/Dockerfile

```diff
@@ -1,12 +1,14 @@
-FROM alpine:latest as certs
-RUN apk add --update --no-cache ca-certificates
+ARG base_image
+ARG debug_image
 
-FROM scratch
+FROM $base_image AS release
 ARG TARGETARCH=amd64
-
-COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
-
-EXPOSE 16686
 COPY query-linux-$TARGETARCH /go/bin/query-linux
-
+EXPOSE 16686/tcp
 ENTRYPOINT ["/go/bin/query-linux"]
+
+FROM $debug_image AS debug
+ARG TARGETARCH=amd64
+COPY query-debug-linux-$TARGETARCH /go/bin/query-linux
+EXPOSE 12345/tcp 16686/tcp
+ENTRYPOINT ["/go/bin/dlv", "exec", "/go/bin/query-linux", "--headless", "--listen=:12345", "--api-version=2", "--accept-multiclient", "--log", "--log-output=debugger,debuglineerr,gdbwire,lldbout,rpc", "--"]
```

Comment:
I think we can make it less verbose: people debugging a Jaeger component would be interested mostly (or only) in the Jaeger output.